packages/flutter_tools/lib/src/device.dart

```diff
@@ -269,8 +269,8 @@ abstract class Device {
 
     // Join columns into lines of text
     return table.map((List<String> row) =>
-        indices.map((int i) => row[i].padRight(widths[i])).join(' • ') +
-        ' • ${row.last}');
+        indices.map((int i) => row[i].padRight(widths[i])).join(' - ') +
+        ' - ${row.last}');
```

Comment:
Flutter is all-in on using the prettiest and most awesomest Unicode in its messages.